puzzles/models.py

```diff
@@ -41,10 +43,20 @@ class PuzzleTag(TagBase):
         (WHITE, "white"),
         (BLACK, "black")
     ]
+    COLOR_ORDERING = {
+        RED: 0,
+        BLACK: 1,
+        WHITE: 2,
+        GRAY: 3,
+        BLUE: 4,
+        GREEN: 5,
+        YELLOW: 6
+    }
     color = models.CharField(
         max_length=10,
         choices=COLORS,
         default=BLUE)
+    is_meta = models.BooleanField(default=False)
```

Comment:
I suggest adding a comment that this is an internal field not visible or modifiable from the UI.